### PR TITLE
update some test util

### PR DESCRIPTION
**Pull request by YangKian**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -169,3 +169,12 @@ sourceSets {
...
 }
+
+test {
+    if (System.getenv('CONFIGS_FILE') == null) {
+        def rootDir = project.rootDir
+        environment('CONFIGS_FILE', "${rootDir}/local/config.yaml")
+    }
+
+    System.out.println("Using CONFIGS_FILE environment variable: " + System.getenv('CONFIGS_FILE'))
+}
```
Comment (4eUeP) [inline app/build.gradle]: config file is optional, so there should not be a default value.

File: app/src/test/scala/utils/kafka/KafkaServerTestHarness.scala [hunk 2]
```diff
@@ -130,10 +131,34 @@ abstract class KafkaServerTestHarness extends QuorumTestHarness {
...
+    // if (code != 0) {
+    //  throw new RuntimeException(s"Failed to delete logs, exit code: $code")
+    // }
+
+    // Delete all metastore(zk) nodes
+    val metastorePort = config.testingConfig
+      .getOrElse("metastore_port", throw new IllegalArgumentException("metastore_port is required"))
+      .asInstanceOf[Int]
+    s"docker run --rm --network host zookeeper:3.8 zkCli.sh -server 127.0.0.1:$metastorePort deleteall /hstream".!
+  }
```
Comment (4eUeP) [inline app/src/test/scala/utils/kafka/KafkaServerTestHarness.scala]: This should be in `shutdownServers`

**Review by 4eUeP — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 1 above)
Comment (YangKian) [reply app/build.gradle]: 1. effective for testing only
2. if you don't need to use this environment variable, then exporting it additionally has no other effect
3. adding this variable can be very convenient in scenarios such as using ide and executing multi tests together.

File: app/src/test/scala/utils/kafka/KafkaServerTestHarness.scala (hunk 2 above)
Comment (YangKian) [reply app/src/test/scala/utils/kafka/KafkaServerTestHarness.scala]: tearDown() call this first, then call `TestUtils.shutdownServers(_brokers)`

**Pull request merged**